db_scripts/init_db.sh

```diff
@@ -7,7 +7,7 @@ if [ $# -ge 1 ] ; then
 fi
 
 NS=esignet
-CHART_VERSION=0.0.1-develop
+CHART_VERSION=1.7.0-develop
```

Comment:
this is postgres-init version this cant be 1.7.0-develop .

It should be 12.0.1